2021_05_10.md

```diff
@@ -36,5 +36,6 @@ Some of the topics we hit on, in order that we hit them:
 -   Bill Joy, [The Future Doesn't Need Us](https://www.wired.com/2000/04/joy-2/), ca. 2000
 -   [E10K](https://en.wikipedia.org/wiki/Sun_Enterprise#Enterprise_10000), code-named "Starfire"
 -   Tom Lyon and Joseph Skudlarek's USENIX 1985 paper, [All The Chips That Fit](https://drive.google.com/file/d/1jVG5YXw_1JNlDwpbLi58dW1dbEQLON3M/view)
+-   [Happy Meal Ethernet](http://www.ozguru.mu.nu/archives/2005/01/happy_meal_ethe.html)
 
-Our next Twitter Space will be on May 17th, 2021 at 5p Pacific!
```

Comment:
I'm not sure why git seems to think there is a diff here. It goes away when viewed while hiding whitespace changes.

Reply:
probably a newline/not at the end of line